pkg/registry/networking/clustercidrconfig/strategy_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clustercidrconfig
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/kubernetes/pkg/apis/networking"
+)
+
+func newClusterCIDRConfig() networking.ClusterCIDRConfig {
+	return networking.ClusterCIDRConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "foo",
+		},
+		Spec: networking.ClusterCIDRConfigSpec{
+			IPv4: &networking.CIDRConfig{
+				CIDR:            "10.1.0.0/16",
+				PerNodeMaskSize: int32(24),
+			},
+			IPv6: &networking.CIDRConfig{
+				CIDR:            "fd00:1:1::/64",
+				PerNodeMaskSize: int32(120),
+			},
+			NodeSelector: &metav1.LabelSelector{MatchLabels: map[string]string{"foo": "bar"}},
+		},
+	}
+}
+
+func TestClusterCIDRConfigStrategy(t *testing.T) {
+	ctx := genericapirequest.NewDefaultContext()
+	apiRequest := genericapirequest.RequestInfo{APIGroup: "networking.k8s.io",
+		APIVersion: "v1",
```

Comment:
I assume this doesn't influence the test,

Reply:
Updated to v1alpha1 but seems like doesn't influence the test